source/extensions/filters/network/thrift_proxy/router/upstream_request.h

```diff
@@ -71,15 +71,20 @@ struct UpstreamRequest : public Tcp::ConnectionPool::Callbacks,
   ProtocolPtr protocol_;
   ThriftObjectPtr upgrade_response_;
 
+  enum class ResponseState : uint8_t {
+    ResponseNonStarted = 0,
+    ResponseStarted = 1,
+    ResponseCompleted = 2,
+    ConnectionReleased = 3
+  };
```

Comment:
nit: I don't think we need to repeat Response on every enum member, so maybe something like:

```
  enum class ResponseState : uint8_t {
    NotStarted = 0,
    Started = 1,
    Completed = 2,
    ConnectionReleased = 3
  };
```

I don't like `NotStarted` that much, maybe `None` is better?

Reply:
Fixed. Thanks!